### PR TITLE
Checking enemy mechanics fully

### PR DESCRIPTION
**Pull request by MysticAmazon**
some things will have to happen after jeff is finished with the enemy codes

### CODE REVIEW

**Review by Jeff-Moreau — CHANGES_REQUESTED** [review]
put the koala stuff back in :P

File: Source/SteveSloth/Private/MyKoalaEnemyClass.cpp
Comment (Saretzky) [inline Source/SteveSloth/Private/MyKoalaEnemyClass.cpp:1]: does this need to be commented out?

**Review by Saretzky — CHANGES_REQUESTED** [review]

**Review by CrazyTobi — CHANGES_REQUESTED** [review]
Should uncomment the koala script for the time being

**Review by Saretzky — APPROVED** [review]
looks good

**Review by CrazyTobi — APPROVED** [review]
Looks good

**Pull request merged**